stepper.c

```diff
@@ -683,13 +683,10 @@ void keyme_init()
   ESTOP_PORT &= ~(ESTOP_BIT);           //estop input normal-low
 
   // Microstepping
-
-  #ifdef SPI_STEPPER_DRIVER
-    /* TODO: Initialize motor drivers with tables here */
-  #else
+  if (settings.spi_motor_drivers == 0) {
```

Comment:
Seems a little odd that the motors have bits and pieces of init code laying around different places in the codebase.  We should probably think about writing a motor driver abstraction layer (or expanding motor_drv) and consolidating this stuff.